indico/modules/events/surveys/templates/submission_preview.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'events/surveys/manage_survey_base.html' %}
+{% from 'forms/_form.html' import form_row %}
+
+{% set survey = submission.survey %}
```

Comment:
The problem was with the base template which uses `survey`. I can simply use `forms/_base.html`